aces/Loris
# [configuration] adds confirmation prompt when deleting a field.

## Brief summary of changes

Adds confirmation prompt when deleting a field.
See more details on issue #7010 

Test Plan updated.

#### Testing instructions

1. Go to `MainMenu-> Admin -> Configuration -> Statistics.`
2. Click on the **"X"** button for an already submitted(existing) field.
3. A dialog box should ask for delete confirmation before proceeding.

#### Link to related issue

* Resolves #7010
